utah-design-system-website/src/react/components/websiteContent/library/components/containers/TabGroup/TabGroupExampleCodeReact.jsx

```diff
@@ -0,0 +1,42 @@
+import { ExampleCodeReactProp } from '../../../../../sandbox/ExampleCodeReactProp';
+
+/** @typedef {import('utah-design-system-website').TabGroupExamplePropsShape} TabGroupExamplePropsShape */
+
+/**
+ * @param {object} props
+ * @param {object} props.state
+ * @param {TabGroupExamplePropsShape} props.state.props
+ * @returns {import('react').JSX.Element}
+ */
+
+export function TabGroupExampleCodeReact({
+  state: {
+    props: {
+      tabA,
+      tabB,
+      panelA,
+      panelB,
+      isVertical,
+    },
+  },
+}) {
+  return (
+    <>
+      &lt;TabGroup
+      <ExampleCodeReactProp displayProp={` isVertical="${isVertical}">`} indentLevel={0} />
+      <ExampleCodeReactProp displayProp="<TabList>" indentLevel={1} />
+      <ExampleCodeReactProp displayProp={`<Tab id="button-A">${tabA}</Tab>`} indentLevel={2} />
```

Comment:
maybe instead of `button-A` it could be `tab-A`? Might have to change the render and a couple of places for this, but it caught me by surprise to see button for a tab.